src/app/progress/milestone.scss

```diff
@@ -146,6 +146,40 @@
   }
 }
 
-.well-rested {
-  cursor: none;
+.well-rested,
+.seasonal-rank {
+  cursor: default;
+}
+
+.seasonal-rank {
+  .milestone-icon {
+    max-width: calc(var(--item-size) * 1.2);
+  }
+
+  &.has-premium-rewards {
```

Comment:
One idea would be to display the non-premium reward as a smaller overlay over the premium reward, rather than displaying them side-by-side.